tests/forte/data/data_pack_test.py

```diff
@@ -170,6 +172,59 @@ def test_get_entries(self):
             ],
         )
 
+        # Case 3: Get Child entries when fetching parents with include_sub_type
+        all_anno: List[Tuple[int, str]] = []
+        for doc in self.data_pack.get(Document):
+            for anno in self.data_pack.get(Annotation, doc):
+                # Just to make the test case concise,
+                # We will only consider Sentence, EntityMention
+                # and PredicateArgument
+                if any(
+                    isinstance(anno, check)
+                    for check in [Sentence, EntityMention, PredicateArgument]
+                ):
+                    all_anno.append(anno.text)
```

Comment:
In order for this test to be readable, I suggest we do `all_anno.append(check.__name__, anno.text, anno.begin, anno.end)`, we can actually see whether the return list complies to the order we expect.